### PR TITLE
fixes: #343 Calendar looks more like wireframes

### LINKED ISSUE #343: Make calendar look more similar to wireframes
## Summary

- [ ] Make map look more similar to wireframe spec

## Motivation

Prepare for 4/15 launch

### PR DESCRIPTION
**Pull request by crispinamuriel**
fixes #343 

### Please provide enough information so that others can review your pull request:
I've styled the calendar so the text is white and the background is a gradient darkblue => blue

### Explain the **details** for making this change. What existing problem does the pull request solve?  
The way I did this was by importing the react-calendar module main CSS file called Calendar.css and make the changes there. This CSS file is now located in ```/client/src/css/Calendar.css```

### Test plan (required)

Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.
Screenshot attached
Screen Shot 2020-04-04 at 3 58 32 PM

### Final Checklist

- [ x] Have you bumped the version in `package.json`?
    - Second decimal for major change, third decimal for minor change. This can go past 10 i.e. 1.0.9 !=> 1.1.0, 1.0.9 => 1.0.10
    - [Read here for more info](https://semver.org/)
- [ ] Have you added any new tests necessary?
- [x ] Is your PR rebased off the most current master?
- [ x] Have you squashed all commits, or if you will merge will you squash all commits?
- [x ] Did you use yarn, not npm?
- [x ] Did you use Material-UI wherever possible?
- [x ] Did you format according to Prettier?
- [x ] Did you run all of your most recent changes locally to make sure everything is working?

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (AdhamAH) [inline yarn.lock:4]: I think this should be removed

**Comment by AdhamAH** [comment]
It looks great! I bet it took some time so thank you!

File: yarn.lock (hunk 1 above)
Comment (crispinamuriel) [reply yarn.lock:4]: let me get at it, thanks!

**Comment by crispinamuriel** [comment]
I believe all the yarn.locks are gone, is there a reason why it wasn't caught by .gitignore?

**Comment by AdhamAH** [comment]
Well, I am not sure, to be honest 😅 I think it got to be with the git past comment maybe, it was not added and when you rebase something goes wrong. Git is wired sometimes
Look at [this](https://github.com/COVID-19-electronic-health-system/Corona-tracker/blob/master/CONTRIBUTING.md) you may find something regarding it

**Comment by SomeMoosery** [comment]
Looks like we deleted the correct yarn.lock in the client directory and created one in the root directory (shouldn’t be there). Not a big deal, I can make a quick HOTFIX when I get home later today. 

Just for future reference, we want to keep the yarn.lock from within the /client directory - if a yarn.lock shows up in the root directory, that probably means at one point you ran yarn install in the root directory, not the client directory 👍

**Comment by SomeMoosery** [comment]
Changes look great though!

**Comment by AdhamAH** [comment]
@SomeMoosery yeah I didn't notice it is not inside /client 😅 Sorry about it

**Comment by crispinamuriel** [comment]
ah, i understand, i yarn installed in the wrong directory! my bad.

**Pull request merged**